tensorboard/webapp/metrics/views/card_renderer/scalar_card_container.ts

```diff
@@ -517,4 +520,13 @@ export class ScalarCardContainer implements CardRenderer, OnInit, OnDestroy {
       };
     });
   }
+
+  onSelectTimeChanged(linkedTime: LinkedTime) {
```

Comment:
nit: consider naming the input to "newLinkedTime" or "newValue" to make it more meaningful

see 
https://cs.opensource.google/tensorflow/tensorboard/+/master:tensorboard/webapp/metrics/views/right_pane/settings_view_container.ts;l=195?q=%22onSelectTimeChanged%22&ss=tensorflow%2Ftensorboard

Reply:
Yea that makes sense I renamed it newLinkedTime. This should probably be done in the histogram_card_container as well but, out of scope of this PR.